lib/rails_erd/domain/attribute.rb

```diff
@@ -88,6 +88,11 @@ def inheritance?
         @model.inheritance_column == name
       end
 
+      # Method allows false to be set as an attributes option when making custom graphs.
+      # Its functionality is implemented in the filtered_attributes method.
+      def false?
+      end
```

Comment:
My concern is that we shouldn't add a method that we don't understand the purpose of. Maybe we can mob on this sometime this week and trace it down?